### PR TITLE
Added Deployment and Validation Scripts for RDA and ARD

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: proposals/dao/feiRari.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const run: RunUpgradeFunc = async (addresses, oldContracts, contracts, logging = false) => {}
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {};
+
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts) => {
+    const { autoRewardsDistributor, rewardsDistributorAdmin } = contracts;
+    const AUTO_REWARDS_DISTRIBUTOR_ROLE = await rewardsDistributorAdmin.AUTO_REWARDS_DISTRIBUTOR_ROLE();
+
+    /// check that the contracts were wired together properly
+    expect(await rewardsDistributorAdmin.hasRole(AUTO_REWARDS_DISTRIBUTOR_ROLE, autoRewardsDistributor.address)).to.be.true;
+    expect(await autoRewardsDistributor.rewardsDistributorAdmin()).to.be.equal(rewardsDistributorAdmin.address);
```
Comment (Joeysantoro) [inline proposals/dao/feiRari.ts:17]: add a check that the OA timelock `isContractAdmin()` on the RDA
also check the rewardsDistributorContract on RDA is the Delegator

check `getNewRewardSpeed()` is correct for ARD, should be around 24 TRIBE/block

**Review by Joeysantoro — COMMENTED** [review]
We should add         _setContractAdminRole(keccak256("TRIBAL_CHIEF_ADMIN_ROLE"));
to the ARD too in the constructor

So far looks good other than missing a few checks

File: proposals/dao/feiRari.ts (hunk 1 above)
Comment (ElliotFriedman) [reply proposals/dao/feiRari.ts]: Added `isContractAdmin` check.

I need to add a check for the second one

Comment (Joeysantoro) [reply proposals/dao/feiRari.ts]: Last one can prob do without

**Review by Joeysantoro — APPROVED** [review]

**Review by Joeysantoro — DISMISSED** [review]

**Review by Joeysantoro — APPROVED** [review]

File: proposals/dao/feiRari.ts (hunk 1 above)
Comment (ElliotFriedman) [reply proposals/dao/feiRari.ts]: Added a check for it.

**Pull request merged**